frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -310,10 +329,13 @@ const determineAvailableFilters = (initialFilters, items, filterGroups) => {
   return filters;
 };
 
-const getActiveFilters = (keywordFilter, groupFilters, activeFilters) => {
+const getActiveFilters = (keywordFilter, groupByFilter, groupFilters, activeFilters) => {
   activeFilters.keyword.value = keywordFilter || '';
   activeFilters.keyword.active = !!keywordFilter;
 
+  activeFilters.groupBy.value = groupByFilter || '';
+  activeFilters.groupBy.active = !!groupByFilter;
```

Comment:
I'm confused. Group by does not actually filter anything, it only affects the visual appearance of the items.
This should not be mixed in with the filters on the page.

Reply:
@christianvogt Initially the ask was to display only the Installed Operators in groups. Later on when we decided to add the Non operators grouped together I should have updated the variable names which I missed

Reply:
moved it out of activeFilters